package.json

```diff
@@ -46,7 +46,7 @@
     "lint": "biome check .",
     "lint-fix": "biome check --write .",
     "lint-css": "stylelint '**/*.css' --ignore-path .gitignore",
-    "tsc": "conc -r 'tsc --noEmit' 'tsc --noEmit -p website/tsconfig.json'",
+    "tsc": "tsc -b",
```

Comment:
Do we need to `--build` here if we're just doing a type check?

Reply:
AFAIK tsc always builds unless noEmit is passed, and I think that flag (build) changes the "mode" or something like that. I don't remember now, but it might have had something to do with incremental mode, or composite perhaps.

Reply:
Yes, this is for [smart incremental builds](https://www.typescriptlang.org/docs/handbook/project-references.html#build-mode-for-typescript). It _should_ make things a little snappier, but also allows specifying individual configs, should you need to pick just a single workspace, eg:

```
npm run tsc -- packages/ariakit-solid-core
```